book/website/reproducible-research/overview/overview-barriers.md

```diff
@@ -0,0 +1,155 @@
+# Barriers to reproducibility
+
+So far we have [explained what we mean](overview-definitions) by reproducible research and explained some of the [additional benefits](overview-benefits).
+
+But there are many barriers to reproducible research.
+
+You can watch Kirstie Whitaker describe some of them in [her talk about _The Turing Way_](https://youtu.be/wZeoZaIV0VE?t=312) at [csv,conf,v4](https://csvconf.com/2019) in May 2019.
+You can use and re-use her slides under a CC-BY licence via Zenodo (doi: [10.5281/zenodo.2669547](https://doi.org/10.5281/zenodo.2669547)).
+The section describing the slide below starts around 5 minutes into the video.
+
+```{figure} ../../figures/barriers-reproducibility.png
+---
+width: 500px
+name: reproducibility-barriers
+alt: "Slide from presentation showing the different barriers to reproducibility. The text in the center says "Barriers to reproducible research" and the following barriers are arranged clockwise around the slide: Is not considered for promotion, Held to a higher standard than others, Publication bias towards novel findings, Plead the 5th, Takes time, Support additional users, Requires additional skills."
```

Comment:
```suggestion
alt: "Slide from the presentation showing the different barriers to reproducibility. The text in the centre says "Barriers to reproducible research" and the following barriers are arranged clockwise around the slide - Is not considered for promotion, Held to a higher standard than others, Publication bias towards novel findings, Plead the 5th, Takes time, Support additional users, Requires additional skills."
```

Reply:
':' and apostrophe (") has a meaning in Sphinx syntax, which is why it was not rendering. This looks good to me besides that error. Checked locally and works with ':' remove.

Reply:
Aaaaah - well done!!! 🌟 🌟 🌟 🌟 🌟  Thank you!